repository/src/main/java/com/epam/esm/repository/impl/CertificateRepositoryImpl.java

```diff
@@ -118,4 +119,24 @@ public Long count() {
                 .getSingleResult();
     }
 
+    private <T> void prepareSearchQuery(AbstractQuery<T> query, Root<Certificate> root, List<String> tagNames, String name) {
+        CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+        List<Predicate> predicates = new ArrayList<>();
+        if (!StringUtils.isBlank(name)) {
```

Comment:
it is better to use the StringUtils.isNotBlank method

Reply:
Right, thanks!